tests/unit/batcomputer_cli/commands/test_identify.py

```diff
@@ -1,12 +1,32 @@
+import re
 import pytest
 
 from ....batcomputer_cli.commands.identify import identify
 from ....batcomputer_cli.database import CHARACTERS
 
 NON_IDENTIFIED_CHARACTERS = ["riddler", "nightwing", "penguin"]
 
+UPPER_CASE_CHARACTERS = [character.upper() for character in CHARACTERS]
+TITLE_CASE_CHARACTERS = [character.title() for character in CHARACTERS]
 
-@pytest.mark.parametrize("alias", CHARACTERS)
+@pytest.mark.parametrize(
+    "alias", list(CHARACTERS) + UPPER_CASE_CHARACTERS + TITLE_CASE_CHARACTERS
+)
+def test_identify_with_one_identified_alias_and_its_oracle(
+    alias: str, capsys: pytest.CaptureFixture[str]
+) -> None:
+    bring_oracle = True
+    identify([alias], bring_oracle)
+    output, error = capsys.readouterr()
+    print("output", output)
+    assert not error
+    assert "A.K.A" in output
+    assert re.search('(\\d{4})', output)
```

Comment:
what this does?

Reply:
Something like
```py
assert "The Dark Knight (2008)" in output
```
would not be simpler?

And with this test, you have finish the TODO in line 50 of this file
```py
# TODO test with oracle option
```
Can you please delete it?

Reply:
> what this does?

It's a regex to match the result. But I also can change to `assert "The Dark Knight (2008)" in output` as suggested.